sw/tests/testPeripheralsGating.c

```diff
@@ -0,0 +1,39 @@
+// Copyright 2024 ETH Zurich and University of Bologna.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Lorenzo Leone <lleone@iis.ee.ethz.ch>
+
```

Comment:
Add a comment that this does not automate testing :)

Reply:
Done it